plugins/Toolbox/src/Toolbox.py

```diff
@@ -169,6 +169,27 @@ def _onAppInitialized(self) -> None:
             "materials_showcase": QUrl("{base_url}/showcase".format(base_url=self._api_url))
         }
 
+    OLD_PLUGINS = ['3DPrinterOS',
+                   'BarbarianPlugin',
+                   'CuraSolidWorksPlugin',
+                   'MakePrintablePlugin',
+                   'OctoPrintPlugin',
+                   'OrientationPlugin',
+                   'ZOffsetPlugin',
+                   'cura-siemensnx-plugin']
+
+    # check for plugins that were installed with the old plugin-browser
+    def _isOldPlugin(self, plugin_id) -> bool:
+        if plugin_id in self.OLD_PLUGINS and plugin_id in self._plugin_registry.getInstalledPlugins():
```

Comment:
Why check in 2 places? Shouldn't only checking the plugin registry be enough?

Reply:
If it is in the plugin_registry I assume it has been loaded correctly.

Reply:
But why do we need `OLD_PLUGINS`? If it's in the plugin registry that should be enough right?